pkg/cvo/cvo_scenarios_test.go

```diff
@@ -59,7 +59,11 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 			} else {
 				existing.Spec = obj.Spec
 				existing.ObjectMeta = obj.ObjectMeta
-				obj.Generation++
```

Comment:
`obj` is the object received in the request payload. The generation is incremented here but this change is not stored back into the `existing` object. So removed the line and changed logic so the highest value of generation is used and then incremented.